### PR TITLE
Display error instead of just say command is incorrect

### PR DESCRIPTION
**Pull request by mhmxs**

### CODE REVIEW

**Comment by jszwedko** [comment]
:+1: thanks @mhmxs! This seems reasonable to me. Do you mind making the analogous change in `app.go` and adding a couple of tests that assert this behavior?

File: app.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func (a *App) Run(arguments []string) (err error) {
 			HandleExitCoder(err)
 			return err
 		}
-		fmt.Fprintf(a.Writer, "%s\n\n", "Incorrect Usage.")
+		fmt.Fprintf(a.Writer, "%s %s\n\n", "Incorrect Usage.", err.Error())
```
Comment (mhmxs) [inline app.go]: Covered by test

File: app.go [hunk 2]
```diff
@@ -315,7 +315,7 @@ func (a *App) RunAsSubcommand(ctx *Context) (err error) {
...
```
Comment (mhmxs) [inline app.go]: New test added

File: command.go [hunk 3]
```diff
@@ -138,7 +138,7 @@ func (c Command) Run(ctx *Context) (err error) {
 			HandleExitCoder(err)
 			return err
 		}
-		fmt.Fprintln(ctx.App.Writer, "Incorrect Usage.")
+		fmt.Fprintln(ctx.App.Writer, "Incorrect Usage:", err.Error())
```
Comment (mhmxs) [inline command.go]: Covered by test

File: app.go (hunk 2 above)
Comment (mhmxs) [inline app.go]: New test added

**Comment by mhmxs** [comment]
@jszwedko plz review

**Comment by jszwedko** [comment]
Looks great, thank you for adding the tests!

**Pull request merged**